core/src/main/scala/kafka/controller/ControllerChannelManager.scala

```diff
@@ -426,15 +426,15 @@ class ControllerBrokerRequestBatch(controller: KafkaController) extends  Logging
       case e : Throwable => {
         if (leaderAndIsrRequestMap.size > 0) {
           error("Haven't been able to send leader and isr requests, current state of " +
-              s"the map is $leaderAndIsrRequestMap")
+              s"the map is $leaderAndIsrRequestMap. Exception message: $e")
         }
         if (updateMetadataRequestMap.size > 0) {
           error("Haven't been able to send metadata update requests, current state of " +
-              s"the map is $updateMetadataRequestMap")
+              s"the map is $updateMetadataRequestMap. Exception message: $e")
         }
         if (stopReplicaRequestMap.size > 0) {
           error("Haven't been able to send stop replica requests, current state of " +
-              s"the map is $stopReplicaRequestMap")
+              s"the map is $stopReplicaRequestMap. Exception message: $e")
         }
         throw new IllegalStateException(e)
```

Comment:
I was wondering the same thing. In a number of cases, it does. Haven't checked the more complex cases though. Also, with this code, would we be including the exception message multiple times if more than once `if` case is true?